### DISCUSSION TITLE
Windows Service - fails to auto start

### DISCUSSION THREAD

**Discussion by hl2guide** [AdguardTeam/AdGuardHome#4422] [Q&A]
I've installed AdGuard Home in Windows 10 21H2 as a Windows Service.

I used:

`AdGuardHome.exe -s install`

It starts and stops manually and functions fine but the auto-start does not work and just shows as Stopped on Windows startup.

Anyone run into this?

**Comment by cnzzr** [comment]
check service Adgurdhome status is auto
